cmd/es-rollover/app/init/flags_test.go

```diff
@@ -34,20 +34,20 @@ func TestBindFlags(t *testing.T) {
 	v.BindPFlags(command.PersistentFlags())
 
 	err := command.ParseFlags([]string{
-		"--shards=8",
-		"--replicas=16",
-		"--priority-span-template=300",
-		"--priority-service-template=301",
```

Comment:
this is a breaking change. We can do breaking changes, but we need to follow the policy: https://github.com/jaegertracing/jaeger/blob/10571d97aad122f42b36801b2e4e28067642b308/CONTRIBUTING.md?plain=1#L193

Reply:
I actually have an alternative suggestion. Instead of doing breaking changes to CLI flags and then supporting deprecated flags, you can roll back the cli flags changes altogether. Let's look at the changes differently:
  * in v2 you divided settings  into 4 distinct groups, allowing more flexibility -- useful change
  * you then pushed that config changes into the implementation of storage components -- also useful change because we keep using the same code with the new config
  * you propagated the options separation into CLI flags -- a change that was not necessary. Since we plan to deprecate v1 next year anyway, we could just keep all v1 CLI flags as they are today. In cases where there are now 4 parameters instead of formerly 1 parameter, you can still have a single CLI flag and just clone its value into all 4 config fields (which would be completely backwards compatible behavior). I think this will minimize the changes in this PR and will not require us introducing breaking changes, supporting deprecated flags, etc.

Reply:
Ok, just like I'm dealing with the same logic in `/plugin/storage/es/options.go`

https://github.com/jaegertracing/jaeger/blob/9de6040aa6ed236e2a530dbe850bb9a048b32e16/plugin/storage/es/options.go#L379C40-L379C63 

I'll update it～